switchbot_mqtt/__init__.py

```diff
@@ -27,6 +27,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+availability_topic="switchbot_mqtt/availability"
```

Comment:
Maybe we should use `/status` instead of `/availability` to follow home assistant's convention?
https://web.archive.org/web/20220829100223/https://www.home-assistant.io/docs/mqtt/birth_will/